src/hub/Hub.sol

```diff
@@ -233,7 +233,8 @@ contract Hub is Multicall, Auth, Recoverable, IHub, IHubGatewayHandler {
         (uint128 approvedAssetAmount,) =
             shareClassManager.approveDeposits(poolId, scId, maxApproval, paymentAssetId, valuation);
 
-        sender.sendApprovedDeposits(poolId, scId, paymentAssetId, approvedAssetAmount);
+        // TODO: Fix price once SCM refactor is in
```

Comment:
Reminder for these open TODOs

Reply:
Done in https://github.com/centrifuge/protocol-v3/pull/306